src/app/models/internationalPayment.ts

```diff
@@ -0,0 +1,14 @@
+export interface InternationalPayment {
+  cardno: number;
+  cvv: string;
+  expirymonth: string;
+  expiryyear: string;
+  amount: number;
+  billingzip: string;
+  billingcity: string;
+  billingaddress: string;
+  billingstate: string;
+  billingcountry: string;
+  save_card: boolean;
```

Comment:
Hey @patrickf949, please ensure to fix tslint issues.

Reply:
Thanks @CryceTruly done

Reply:
Great eye. @CryceTruly, however, since these values are passed to the backend I have to use the convention we chose to use for the backend